### PR TITLE
Add devise

### PR DESCRIPTION
**Pull request by tomasesquivelgc**
# In this branch my partner @Marcraphael12 and I:
 - Add the devise Gem to the project
 - Create the user validation
 - Add a Navbar that allows to login, logout, or signup depending on the current_user

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (Shedrack-Sunday) [inline app/models/user.rb]: - [x] As part of the project requirement,  `: confirmable` pick should be enabled in device  in the user model (user.rb file) which is not so here for you. Kindly see to it👍 

### Signup without confirmation:
image


### Instruction:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @tomasesquivelgc  @Marcraphael12     👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with Rails models and setting device✔️
- Kudus making the requirement asked      ✔️  

# - Required Changes  🌵  

- [ ] Kindly take note of the comments i made in the regarding  confirmation of signup in your app   👍  

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Shedrack-Sunday** [comment]
Hello @tomasesquivelgc @Marcraphael12 , I mistakenkly approved your project on the dashboard, as you can see you need to make changes above. Please make the required change, while the review team fixes this. My sincere apologies 🙏

**Comment by enniopo** [comment]
Hi @tomasesquivelgc - the activity's status has been updated now. Please, request another code review request when you are ready. Thank you

**Review by cynthiainga — APPROVED** [review]
Hi @tomasesquivelgc, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] User can confirm their email ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**